activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -28,6 +31,15 @@ def self.extended(base) # :nodoc:
       end
     end
 
+    # Declares a method that makes it possible to extend components of Rails
+    # after they are fully loaded.
+    #
+    # Beware that if a component of Rails is extended without using this method
```

Comment:
This sentence is out of place. This seems more a warning in the rails initialization guide than part of the documentation of on load